pycytominer/cyto_utils/cell_locations.py

```diff
@@ -147,7 +147,7 @@ def _s3_file_exists(self, s3_path: str):
         bucket, key = self._parse_s3_path(s3_path)
 
         try:
-            self.s3.Object(bucket, key).load()
+            self.s3.head_object(Bucket=bucket, Key=key)
         except botocore.exceptions.ClientError as e:
```

Comment:
Double checking: does this exception get triggered by `boto3.s3.head_object()`?

Reply:
yes